packages/pluggableWidgets/video-player-native/src/VideoPlayer.tsx

```diff
@@ -1,11 +1,22 @@
 import { flattenStyles } from "@mendix/piw-native-utils-internal";
-import { Component, createElement } from "react";
-import { ActivityIndicator, Text, View } from "react-native";
-import Video, { OnLoadData } from "react-native-video";
+import { createElement, Fragment, ReactElement, useCallback, useEffect, useRef, useState } from "react";
+import {
+    ActivityIndicator,
+    Modal,
+    Platform,
+    StatusBar,
+    Text,
+    TouchableOpacity,
+    TouchableWithoutFeedback,
+    View
+} from "react-native";
+import Video, { OnProgressData, VideoProperties } from "react-native-video";
+import SystemNavigationBar from "react-native-system-navigation-bar";
+import Icon from "react-native-vector-icons/MaterialIcons";
 import { VideoPlayerProps } from "../typings/VideoPlayerProps";
-import { defaultVideoStyle } from "./ui/Styles";
-
-export type Props = VideoPlayerProps<undefined>;
+import { defaultVideoStyle, VideoStyle } from "./ui/Styles";
+import { isAvailable } from "@mendix/piw-utils-internal";
+import deepmerge from "deepmerge";
```

Comment:
I think `deepmerge` should be included in dependencies list

Reply:
done